tests/unit/test_config.py

```
@@ -0,0 +1,172 @@
+        with patch.dict(os.environ, {"TEST_FLAG": "no"}):
+            result = one_req("TEST_FLAG")
+            assert result is False
+
+
+class TestPrintConfig:
+    """Tests for PrintConfig dataclass."""
+
+    def test_is_frozen(self) -> None:
+        """PrintConfig should be immutable (frozen)."""
+        config = PrintConfig(noprint_formats=True, noprint=False)
+        with pytest.raises(Exception):  # FrozenInstanceError in dataclasses
```

Ruff (0.14.14)

[warning] Do not assert blind exception: `Exception`

(B017)